.github/workflows/github-ci.yml

```
@@ -17,15 +17,26 @@ jobs:
   rebase-check:
     name: Rebase Check
     runs-on: ubuntu-latest
-    if: github.event_name == 'pull_request' || github.event_name == 'push'
+    if: github.event_name == 'pull_request' || github.event_name == 'push' # Le job s'exécute aussi sur push
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)